hazelcast/src/main/java/com/hazelcast/sql/SqlResult.java

```diff
@@ -35,13 +41,32 @@
  * }
  * </pre>
  *
- * @see #iterator()
- * @see #close()
+ * <p>
+ * <h4>Usage for update count</h4>
+ *
+ * <pre>
+ *     long updated = hazelcastInstance.getSql().query("UPDATE ...");
```

Comment:
Looks like this should be changed to `hazelcastInstance.getSql().query("UPDATE ...").updatedCount()`